Sources/NostrSDK/Events/NostrEvent.swift

```diff
@@ -114,4 +124,14 @@ public class NostrEvent: Codable, Equatable {
     public func allValues(forTagName tag: TagName) -> [String] {
         tags.filter { $0.name == tag.rawValue }.map { $0.value }
     }
+
+    func isEqual(to nostrEvent: NostrEvent) -> Bool {
```

Comment:
I'd rather not expose two different methods of checking equality, and the `isEqual(to:)` pattern feels like a lot of Apple's legacy Objective-C stuff. Can you just move all of this function into the `==`?

Reply:
Great feedback. I thought it was weird but tried to be consistent since other classes followed this pattern. I've moved it into `==`, and I've done the same for other classes as well.